src/tm_devices/commands/_1mlt9u_mdomsodpo/measurement.py

```diff
@@ -2767,6 +2815,51 @@ def minimum(self) -> MeasurementMeasItemMinimum:
         """
         return self._minimum
 
+    @property
+    def source1(self) -> MeasurementMeasItemSource1:
+        """Return the ``MEASUrement:MEAS<x>:SOUrce1`` command.
+
+        **Description:**
+            - For SOURce1: This command specifies the source for all single channel measurements.
+              For delay or phase measurements, This command specifies the waveform to measure
+              'from'. This is equivalent to setting the 'From:' waveform in the 'Measure Delay' side
+              menu or the 'Measure Phase' side menu. SOUrce is equivalent to SOURCE1. For SOUrce2:
```

Comment:
Why is this referencing SOUrce2 when source 2 isn't available? The syntax appears to be hard-coded to source1.